RELEASING.md

```diff
@@ -259,36 +276,33 @@ as described in the section above.
    make release-publish
    ```
 
-1. Merge the PR. This will cause candidate.docs.projectcalico.org to be updated (after a few minutes). Validate that everything looks correct before proceeding to the next step.
+1. Merge the PR. This will cause docs.projectcalico.org to be updated (after a few minutes). Validate that everything looks correct.
 
-1. Checkout the master branch
+## Adding the previous release to docs.projectcalico.org/archive
 
-1. In [netlify.toml](netlify.toml):
-
-   1. Set the `CANDIDATE_RELEASE` environment variable back to an empty string.
-
-   1. Update the `CURRENT_RELEASE` environment variable.
+1. Enable branch deployments for previous release branch on netlify by adding branch name to `Deploy contexts` in site settings.
+ 
+1. Checkout to previous release branch. Add a commit to the branch, an empty commit if necessary (to trigger the branch deploy). 
 
-1. In [netlify/_redirects](netlify/_redirects), add a new line for the new release.
+1. Open a pull requests to upstream previous release branch, get it reviewed and merged. This triggers branch deployment for the previous release.
 
-1. Commit your changes. For example:
+1. Ensure branch deployment is done by visiting to `<previous-release-branch-name>--calico.netlify.app`
 
+1. Check out to production branch. 
    ```
-   git commit -m "Promote vX.Y.Z to latest"
+   git checkout release-vX.Y
    ```
 
-1. Push your branch and open a pull request to the upstream master branch. Get it reviewed and wait for it to pass CI.
-
-## Adding the previous release to archive.docs.projectcalico.org
+1. Add a new stanza for proxy for the previous release.
 
-1. Checkout latest master.
-
-   ```
-   git checkout master
+   ```toml
+    [[redirects]]
+      from = "/archive/<previous-release>/*"
+      to = "https://<previous-release-branch>--calico-test.netlify.app/:splat"
+      status = 200
+      force = true
+      headers = {X-From = "Netlify"}
    ```
-
-1. Add the previous release to the top of `_data/archive.yml`
```

Comment:
I think we still need to add the previous release to the top of this file as it is used to generate the list of releases on the /releases page